ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/avd_schema_documentation.j2

```diff
@@ -1,28 +1,27 @@
-!!! warning
-    This document describes the data model for AVD 4.x. It may or may not work in previous versions.
-{% for table in documentation_file.tables %}
+---
+search:
+  boost: 2
+---
 
+# {{ documentation_file.filename.split("/")[-1] }}
+{% for table in documentation_file.tables %}
```

Comment:
if you insert a newline just above the heading in line 8 (inside the loop), it will sort out both newline issues I have commented.